src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img 
+          src="./images/logo.png" 
+          alt="logo" 
+          class="header__logo"
+        >
+      </a> 
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#apple" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__item">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+          <li class="header__item">
```

Comment:
```suggestion
          </li>
          <li class="header__item">
            <a href="#samsung" class="header__link">
              Samsung
            </a>
          </li>
          
          <li class="header__item">
```
fix code style all over the page